requirements.txt

```diff
@@ -0,0 +1,6 @@
+dj-database-url==0.4.1
+Django==1.9
+gunicorn==19.6.0
+pre-commit
```

Comment:
Please move this package to separate file `test_requirements.txt` and source this file with `-r` there.